### ISSUE TITLE
afq.mat file includes scanning parameters

### ISSUE THREAD

**Issue by jyeatman** [yeatmanlab/AFQ-Browser#170]
structured array

**Comment by arokem** [comment]
https://github.com/yeatmanlab/AFQ/pull/20

**Comment by arokem** [comment]
Create a json file that lives in the data folder for this.

**Comment by arokem** [comment]
```
params - |
         |  scan_parameters
         |  afq_parameters
```

**Comment by arokem** [comment]
Data from here: 

https://github.com/yeatmanlab/AFQ-Browser_data/blob/master/AFQ-Browser_example/AFQ-Browser_example.mat

**Comment by arokem** [comment]
Closed through #225
